python/vast/utils/config.py

```diff
@@ -1,29 +1,19 @@
-import argparse
 from dynaconf import Dynaconf, Validator
 
+# TODO: make the path configurable
 CONFIG_FILES = ["config.yaml", "config.yml"]
 
 
-def create(config_files=CONFIG_FILES):
+def create():
     settings = Dynaconf(
-        settings_files=config_files,
+        settings_files=CONFIG_FILES,
         load_dotenv=True,
         envvar_prefix="VAST",
     )
     settings.validators.register(
-        Validator("fabric.logging.console_verbosity", default="debug"),
-        Validator("fabric.logging.file_verbosity", default="quiet"),
-        Validator("fabric.logging.filename", default="vast.log"),
```

Comment:
@mavam I am wondering about the config namespace we should use here. 
- Do we want the config to be the same as for VAST? That would mean that setting VAST_CONSOLE_VERBOSITY=debug sets the debug logging for both the bindings and the binary.
- [VAST config](https://github.com/tenzir/vast/blob/master/vast.yaml.example#L68) uses `-` and we use `_` here, should we unify this?

Reply:
Yes, let's unify where we can.